pkgs/applications/office/ticktick/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, fetchurl
+, stdenv
+, wrapGAppsHook
+, dpkg
+, autoPatchelfHook
+, glibc
+, gcc-unwrapped
+, nss
+, libdrm
+, mesa
+, alsa-lib
+, xdg-utils
+, systemd
+}:
+stdenv.mkDerivation (final: {
```

Comment:
The convention is to name the argument finalAttrs, could you change that ?

Reply:
@TheBrainScrambler To shorten how much extra text was in the strings below, as I was told to convert from a `let ... in` setup to the function method here. I didn't like that I was adding 11 characters to _every_ instance of `${pname}`, it made the lines unnecessarily long. Plus, I'm a new packager and was unaware there was a convention here

Reply:
Yes it's fine if you're unaware about the convention, especially that this one is fairly new, that's why reviews exist ;)

But I don't think shortening text is a good reason. Every package that I've seen using this pattern always named the argument finalAttrs. Consistency is a good thing since if you read someone else's package and see "finalAttrs" you immediately know what it is. If each people instead used their own names, then it will each time be a different one and it makes code hard to read.